src/main/java/lotto/view/OutputView.java

```diff
@@ -19,14 +28,18 @@ public static void printLottos(Lottos lottos) {
 		System.out.println();
 	}
 
-	public static void printResult(GameResultDto gameResult) {
+	public static void printProfit(double profit) {
+		System.out.println("총 수익률은 " + profit + "% 입니다.");
+	}
+
+	public static void printResult(GameResult result) {
 		System.out.println("당첨 통계 \n ==============");
-		printRank(gameResult.getGameResult());
-		printProfit(gameResult.getProfit());
+		printRank(result.getResult().values());
```

Comment:
```suggestion
		printRank(result.getResultCounts());
```

와 같이 수정한다면 어떠한 차이가 있을까요?

Reply:
GameResult에서 가지고 있는 Map에서 Statistic은 외부에서 의미가 없기 때문에 getResultCount()라는 메소드에 values()만을 반환하도록 변경하였습니다. 불 필요한 정보까지 넘기지 않기 때문에 조금 더 명확해진 것 같습니다.